client/css/style.css

```diff
@@ -815,25 +815,30 @@ button {
 #chat .time,
 #chat .from,
 #chat .text {
-	display: table-cell;
+	display: block;
 	padding: 2px 0;
-	vertical-align: top;
+	flex: 0 0 auto;
 }
 
 #chat .time {
 	color: #ddd;
 	text-align: right;
-	max-width: 46px;
-	min-width: 46px;
+	width: 46px;
 }
 
 #chat .from {
 	border-right: 1px solid #f6f6f6;
 	color: #b1c3ce;
 	padding-right: 10px;
 	text-align: right;
-	max-width: 134px;
-	min-width: 134px;
+	width: 134px;
```

Comment:
Same here, :shrug_emoji:
https://github.com/thelounge/lounge/commit/3cddbbce6eca34843c7cc4b876a9ecb12602672e#diff-97db1f70168fb5f12457b238ff6052b5L754

I guess this is why we should really add some comments to our code...